test/parallel/test-internal-errors.js

```diff
@@ -41,15 +41,25 @@ errors.E('TEST_ERROR_2', (a, b) => `${a} ${b}`, Error);
   assert.strictEqual(err.code, 'TEST_ERROR_2');
 }
 
-{
-  assert.throws(
-    () => new errors.codes.TEST_ERROR_1(),
-    {
-      message: 'Code: TEST_ERROR_1; The provided arguments ' +
-               'length (0) does not match the required ones (1).'
-    }
-  );
-}
+// {
+//   assert.throws(() => new errors.codes.TEST_ERROR_1(){
+//      message: 'Code: TEST_ERROR_1; The provided arguments ' +
+//                'length (0) does not match the required ones (1).'
+//     }
+//   );
+// }
+
+assert.expectsError(() => {
```

Comment:
Can you write the tests in the corresponding `test/parallel/test-fs-read*.js` files? Or add a new test for the new error and name it with something like `test/parallel/test-fs-read-empty-buffer.js`.